### PR TITLE
CQ-4350514 Fix DoR related bugs in Adaptive Forms container component…

### PR DESCRIPTION
**Pull request by ujjwal5**
… in Sites page

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/dorhook.js [hunk 1]
```diff
@@ -89,12 +94,19 @@
             return false;
     }
 
+    function fetchAuthorContentFrameDocument() {
+        var contentFrameDocumentArray = ns.ContentFrame ? ns.ContentFrame.getDocument() : [];
+        if (contentFrameDocumentArray && contentFrameDocumentArray.length > 0) {
+            return contentFrameDocumentArray[0];
```
Comment (ujjwal5) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/dorhook.js:100]: ns.ContentFrame.getDocument() and document have two different values.
We are using similar approach to get content frame container in cq-guides too:

https://git.corp.adobe.com/livecycle/cq-guides/blob/master/content/src/main/resources/libs/fd/af/authoring/clientlibs/touchAuthoring/guideTouchAuthoringCommon/javascript/util/TouchUtils.js#L467

I'm not aware of another alternatives, can you please share the same.?

Comment (nit23uec) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/dorhook.js:97]: pls try `editable.path` to get the path of guideContainer directly.

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/dorhook.js [hunk 2]
```diff
@@ -89,12 +89,11 @@
             return false;
     }
 
-    function getGuideContainerProperties() {
-        var formPath = getFormPath(ns.ContentFrame.getContentPath());
+    function getGuideContainerProperties(contentFramePath) {
```
Comment (nit23uec) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/dorhook.js:92]: minor: name of argument is incorrect. 
contentFrame has no path.
it is the path of the editable. we can rename it as `editablePath`

Comment (ujjwal5) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/dorhook.js]: done

**Review by nit23uec — APPROVED** [review]

**Pull request merged**